.travis.yml

```diff
@@ -0,0 +1,39 @@
+sudo: true
+
+language: python
+
+services:
+  - docker
+
+env:
+  -
+    - TOXENV=py27
```

Comment:
I prefer to have multiple Travis targets to get better error reporting and parallel testing. The overhead is minimal. We have to change the file every 18 months to modify three lines. (Python's release cycles is roughly 18 months.)